### PR TITLE
Snapper cleanup test based on FATE#312751

### PR DESCRIPTION
**Pull request by dgutu-suse**

### CODE REVIEW

**Comment by sysrich** [comment]
Do you have a validation run on your local instance?

**Comment by dgutu-suse** [comment]
I've found that I missed the custom timeout for assert_script_run. Let me re-check.

**Comment by okurz** [comment]
Please update the previous PR instead of creating yet another one.

**Comment by sysrich** [comment]
He can't reopen the old PR as the old branch appears to be deleted - reopening this PR because the issue is very important - Oliver, would you be prepared to teach Dumitru a little git-fu about how you'd solve the issue in the future?

**Comment by okurz** [comment]
sure. For review, also look into the previous versions of the same change: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/1405 and https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/1386

**Comment by okurz** [comment]
@dgutu-suse please see my change suggestions in https://github.com/dgutu-suse/os-autoinst-distri-opensuse/compare/features/snapper_cleanup...okurz:feature/snapper_cleanup?expand=1

and my comment in https://progress.opensuse.org/issues/11432

File: products/sle/main.pm [hunk 1]
```diff
@@ -545,6 +545,7 @@ sub load_consoletests() {
             }
             elsif (!get_var("ZDUP") and !check_var('VERSION', '12')) {    # zypper and sle12 doesn't do upgrade or installation snapshots
                 loadtest "console/installation_snapshots.pm";
+                loadtest "console/snapper_cleanup.pm";
```
Comment (okurz) [inline products/sle/main.pm]: shouldn't that be `sle_version_at_least('12-SP2')`?

Comment (dgutu-suse) [inline products/sle/main.pm]: I made changes according to your suggestions and after a couple of run I realized that in my test I missed to check important part of the concept. The quota group must always keep 50% of file system space. I need to add the login of comparison between file system size and free space available after cleaning up the snapshots. Some googling helped me to find and read this - http://snapper.io/2016/05/18/space-aware-cleanup.html 
The test needs to be updated...

**Pull request merged**